src/AccessibilityInsights.SharedUx/Resources/Styles.xaml

```diff
@@ -1229,39 +1229,65 @@
     <Style TargetType="{x:Type CheckBox}" x:Key="CkbxRightSide" BasedOn="{StaticResource CheckBoxContastingBorder}">
         <Setter Property="FontSize" Value="14"/>
     </Style>
-    <Style TargetType="{x:Type MenuItem}" x:Key="miFabIcon">
+    <Style TargetType="{x:Type MenuItem}" x:Key="miBoldOnSelection">
+        <Setter Property="FontWeight" Value="Regular"/>
+        <Style.Triggers>
+            <Trigger Property="IsHighlighted" Value="True">
+                <Setter Property="Background" Value="{DynamicResource ResourceKey=SelectedBrush}"/>
+                <Setter Property="Foreground" Value="{DynamicResource ResourceKey=SelectedTextBrush}"/>
+                <Setter Property="TextBlock.LayoutTransform">
+                    <Setter.Value>
+                        <ScaleTransform ScaleX=".97"/>
```

Comment:
Is it worth adding a comment why this is here? I'm guessing that it's to prevent bold from being too big?

Reply:
Yep, as described in context this makes the bolding wiggle the menu a bit less. Comment added!